lib/elixir/lib/record.ex

```diff
@@ -36,6 +36,10 @@ defmodule Record do
 
   """
 
+  @type type :: :defrecord | :defrecordp
+  @type name :: atom
+  @type tag :: atom
```

Comment:
I have a PR ready after this one gets merged that renamed the arguments and variables called `atom` with `name`, since it is not clear what it is dealing with.

I will add `name :: atom` for the time being